lib/locales/da-DK.yml

```
@@ -565,3 +565,5 @@ da-DK:
           - Bænk
           - Ur
           - Pung
+    vehicle:
+      license_plate: '[A-F]{2}\d{5}'
```

[Wikipedia](https://en.wikipedia.org/wiki/Vehicle_registration_plates_of_Denmark) doesn't mention restricting the letters until F. I imagine we also want to ensure it's a numeric digit:

```suggestion
      license_plate: '[A-Z]{2}[0-9]{5}'
```